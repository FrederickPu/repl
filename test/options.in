{"cmd": "#print List.cons"}

{"cmd": "set_option pp.universes true"}

{"cmd": "#print List.cons"}

{"cmd": "#print List.cons", "env": 0}

