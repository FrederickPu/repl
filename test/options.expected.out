{"results":
 [{"source": "#print List.cons",
   "messages":
   [{"severity": "info",
     "pos": {"line": 1, "column": 0},
     "endPos": {"line": 1, "column": 6},
     "data": "constructor List.cons.{u} : {α : Type u} → α → List α → List α"}],
   "env": 0}]}

{"results": [{"source": "set_option pp.universes true", "env": 1}]}

{"results":
 [{"source": "#print List.cons",
   "messages":
   [{"severity": "info",
     "pos": {"line": 1, "column": 0},
     "endPos": {"line": 1, "column": 6},
     "data":
     "constructor List.cons.{u} : {α : Type u} → α → List.{u} α → List.{u} α"}],
   "env": 2}]}

{"results":
 [{"source": "#print List.cons",
   "messages":
   [{"severity": "info",
     "pos": {"line": 1, "column": 0},
     "endPos": {"line": 1, "column": 6},
     "data": "constructor List.cons.{u} : {α : Type u} → α → List α → List α"}],
   "env": 3}]}

